force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -139,15 +181,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String returnHelloWorld() {
         // Initialize the result String
-
+        String hello;
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
+         for (Integer i = 1; i <3; i++){
+            if(i == 1){
+                hello = 'Hello World!; ';
+            }
+            hello = hello + 'Hello World!; ';
```

Comment:
good job. I think you could have skipped the conditional on line 187 and just built the string.